Formula/gpgme.rb

```diff
@@ -12,7 +13,7 @@ class Gpgme < Formula
     sha256 "4a4c3e90175b02ded974a8a385b6d9799bcc2873c5a1087c558f350eb103934d" => :yosemite
   end
 
-  depends_on "gnupg2"
+  depends_on "gnupg"
```

Comment:
This seems incorrect based on the `inreplace` below? One or the other needs to change.